zerver/lib/actions.py

```diff
@@ -2890,6 +2890,13 @@ def do_set_user_display_setting(user_profile, setting_name, setting_value):
              'setting': setting_value}
     send_event(event, [user_profile.id])
 
+    # set emoji_alt_code according to the choice of emojiset
+    if setting_name == "emojiset":
```

Comment:
This piece of code can be removed once you are done with the `emoji_alt_code` -> `emojiset` refactoring.